### DISCUSSION TITLE
Toggle to directly get PDF view

### DISCUSSION THREAD

**Discussion by bandundu** [LeoKlaus/Paperparrot#109] [Feature Requests]
Hi, 

currently when a document is selected in the app the user by default is redirected to the PDF tag editor. However for some users that only use it as document retrieval it would be helpful to toggle directly jumping into the full pdf view when selected from list.

**Comment by LeoKlaus** [comment]
Hey bandundu, thank you for the suggestion.

I have already received similar feedback and I generally like the idea. I'll see how I can implement this.

**Comment by LeoKlaus** [comment]
I have implemented this with version 1.5.0. I'll publish this update to TestFlight today and plan to publicly release early next week.
